docs/android/release.md

```diff
@@ -43,7 +43,7 @@ You can join the beta program directly through the [Google Play Store](https://p
 
 ### Production release
 
-If the beta version is stable and approved by maintainers, it is promoted to **production**, making it available to all users.
+If the beta version is stable and approved by maintainers, it is promoted to **production**, making it available to all users. The companion docs may need to be updated to remove any beta labels once approved.
```

Comment:
We could have make a checklist maybe. Like

- prepare the changelog in the app
- check release button
- prepare companions docs
- tell Missy about the release and highlights what we wants to say in the social medias

Reply:
done, this comment also made me realize we had other updates to include as part of our release processes